organizing_hub/templates/partials/organizing_hub_header.html

```diff
@@ -26,8 +26,12 @@
             {% if not static_content and request.user.is_authenticated %}
             <div class="col-md-6">
               <div class="uppercase text-right">
+                {% local_group as group %}
+                {% if group %}
```

Comment:
what is the purpose of the `as` statement? why not just put `{% if local_group %}`

Reply:
Since local_group is a template tag you have to declare it as something first, you can't do a traditional if statement on it afaik.